.github/workflows/maven.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will build the m2e project with Maven
+
+name: Build m2e-core
+
+on:
+  push:
+    branches: 
+      - 'master'
+  pull_request:
+    branches: 
+     - 'master'
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Cache local Maven repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: Generated Meta data
+      run: mvn generate-sources -f m2e-maven-runtime -Pgenerate-osgi-metadata -Dtycho.mode=maven
+    - name: Build m2e-core
+      run: mvn clean verify --file pom.xml -fae
```

Comment:
You should use -Dmaven.test.failure.ignore=true instead and publish the test results as done in 
https://github.com/eclipse/tycho/blob/master/.github/workflows/maven.yml#L46
then another action can publish the test results see https://github.com/eclipse/tycho/blob/master/.github/workflows/check.yml

Or simply not use fae if you want to fail fast (what generally is better if we are not talking about integration tests)